### ISSUE TITLE
fs.js issue while bundling; OSX: Yosimite

### ISSUE THREAD

**Issue by lacymorrow** [arboleya/electrify#18]
```
I20151009-17:54:03.332(-4)? INFO  electrify:app:  bundling meteor
W20151009-17:54:03.597(-4)? (STDERR) fs.js:439
W20151009-17:54:03.597(-4)? (STDERR)   return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
W20151009-17:54:03.597(-4)? (STDERR)                  ^
W20151009-17:54:03.598(-4)? (STDERR) Error: ENOENT, no such file or directory '/var/folders/jp/1p1ny1kx2jb3t4jb36ygg7580000gn/T/electrify/bundling/bundle/settings.json'
W20151009-17:54:03.598(-4)? (STDERR)     at Object.fs.openSync (fs.js:439:18)
W20151009-17:54:03.598(-4)? (STDERR)     at Object.fs.writeFileSync (fs.js:978:15)
W20151009-17:54:03.598(-4)? (STDERR)     at ChildProcess.<anonymous> (/Users/lacymorrow/.meteor/packages/arboleya_electrify/.1.4.0.4gvna4++os+web.browser+web.cordova/npm/node_modules/electrify/lib/app.js:71:8)
W20151009-17:54:03.598(-4)? (STDERR)     at ChildProcess.emit (events.js:98:17)
W20151009-17:54:03.599(-4)? (STDERR)     at Process.ChildProcess._handle.onexit (child_process.js:820:12)
```

**Comment by lacymorrow** [comment]
I'll tinker and try the npm method, my app also uses the fs module, could be getting race conditions.

**Comment by lacymorrow** [comment]
NPM method:

```
$ electrify package
INFO  electrify:index:  initializing
INFO  electrify:plugins:  use  mongodb
INFO  electrify:plugins:  use  nodejs
module.js:338
    throw err;
          ^
Error: Cannot find module './package.json'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/electrify/bin/cli.js:11:12)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
```

**Comment by arboleya** [comment]
Hi @lacymorrow, this happens only when packaging the app? When you're just running meteor everything goes fine?

Also, can you tell me if the same occurs with a simple meteor project?

``` shell
meteor create --example leaderboard
cd leaderboard
meteor remove-platform ios android
meteor add arboleya:electrify
meteor
```

**Comment by lacymorrow** [comment]
Hi @arboleya, thanks for getting back with me.

Leaderboard project works fine, it must be something specific to my app.

After running  `meteor add arboleya:electrify`, my project no longer shows up on the local server, just a white empty page with the following chrome console errors:
```
global-imports.js:5 Uncaught TypeError: Cannot read property 'Electrify' of undefined
template.cinematic.js:2 Uncaught ReferenceError: Template is not defined
cinematic.js:38 Uncaught ReferenceError: Mongo is not defined
```

An Electron window launches with an empty white window as well.

In the terminal I see my application start and run normally with expected console output, followed by (what seems like) normal electrify output:
```
I20151010-13:53:15.726(-4)? INFO  electrify:plugins:  acquire
I20151010-13:53:15.726(-4)? INFO  electrify:plugins:mongodb:  acquiring mongo
I20151010-13:53:15.727(-4)? INFO  electrify:plugins:nodejs:  acquiring nodejs
I20151010-13:53:15.727(-4)? INFO  electrify:plugins:  start
I20151010-13:53:15.727(-4)? INFO  electrify:plugins:mongodb:  app in development mode, skipping start
I20151010-13:53:15.729(-4)? INFO  electrify:plugins:nodejs:  app in development mode, skipping start
I20151010-13:53:15.731(-4)? INFO  electrify:app:  ensuring electrify dependencies
I20151010-13:53:16.140(-4)? INFO  electrify:electron:  launching electron
```

Hope this helps! Love your project.

**Comment by arboleya** [comment]
Yes, this is the normal output. Very awkward behavior tough.

I'm working on some improvements, I'll let you know so you can do more tests.

One last question, have you tried to build your project with Meteor alone? It worked?

**Comment by lacymorrow** [comment]
yep, seems to build properly

**Comment by FearlessPixel** [comment]
I have the same issue packaging, but my app runs fine. Same OS

update: same error on my linux machine, and I get the same error packaging the leaderboard and todos example projects

**Comment by mmatvien** [comment]
same error here as well

**Comment by challett** [comment]
Same issue on 64 bit windows

**Comment by arboleya** [comment]
Should be fixed in `2.0.0`, please update and try again.

In case the problem persists, please re-open.

**Comment by lacymorrow** [comment]
Worked like a charm! Thanks @arboleya
